UIComponets/ViewController/UIPageControlVC.swift

```diff
@@ -30,14 +30,14 @@ class UIPageControlVC: BaseViewController {
         updatePageControl()
     }
     // MARK: - IBActions
-    @IBAction func btnNext(_ sender: Any) {
+    @IBAction func btnNextClick(_ sender: Any) {
         if currentPage < pageControl.numberOfPages-1 {
```

Comment:
it should be clicked thoughout the project, some place we have use click and other place we have use clicked